src/internal/analytics/components/analytics-funnel.tsx

```diff
@@ -201,6 +206,19 @@ export const AnalyticsFunnelStep = (props: AnalyticsFunnelStepProps) => {
   return <InnerAnalyticsFunnelStep {...props} key={props.stepNumber} />;
 };
 
+function getSubStepConfiguration() {
+  const subSteps = Array.from(document.querySelectorAll<HTMLElement>(getSubStepAllSelector()));
+
+  const subStepConfiguration = subSteps.map((substep, index) => {
+    const name = substep.querySelector(getSubStepNameSelector())?.textContent ?? '';
```

Comment:
Should we include the sub-steps w/o a name at all?

Reply:
Is there a reason why here we use textContent and in getNameFromSelector we use innerText?

Reply:
Good catch, I don't think there was any particular reason. The advantage of `innerText` is that it doesn't include invisible text. I'll update this to use `innerText`.

Steps without a name should still be included. This only happens for containers that don't have a heading, which we still want to be recognisable in the event data.

Reply:
Should we have a better default than an empty string in that case? Like `name.trim() || null`?